Nie-ontologies/Generic-ontologies/human-geography.ttl

```diff
@@ -76,7 +76,8 @@ humgeo:Government
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/human-geography#>;
 	a rdfs:Class;
 	rdfs:label "government"@en, "Regierung"@de;
-	rdfs:comment """Exercises authority over a particular territory."""@en.
+	rdfs:comment """Group of politicians which exercise authority over a particular territory."""@en.
```

Comment:
"One or more persons in a politician role exercising authority over a particular territory."